### ISSUE TITLE
When there are no Editors/Authors, don't show the respective field

### ISSUE THREAD

**Issue by ronaldtse** [riboseinc/metanorma-csd#70]
Here's how it looks like now with no entry:

![screen shot 2018-12-11 at 4 23 20 pm](https://user-images.githubusercontent.com/11865/49787251-2b01e680-fd61-11e8-9f10-a37b582d32ec.png)

**Comment by opoudjis** [comment]
I can't replicate this, and given the template, I don't see how this is happening:

````
  {% if editors.size > 0 %}
  <div class="coverpage-tc-name">
    <span>Editors: {{ editors | join: ", " }}</span>
  </div>
  {% endif %}
  {% if authors.size > 0 %}
  <div class="coverpage-tc-name">
    <span>Authors: {{ authors | join: ", " }}</span>
  </div>
  {% endif %}
````

The following header:

````
= CalConnect Standard: Calendar operator practices -- Guidelines to protect against calendar abuse
:title: Calendar operator practices -- Guidelines to protect against calendar abuse
:docnumber: 18003
:status: draft-standard
:doctype: report
:edition: 1
:copyright-year: 2018
:revdate: 2018-10-31T00:00:00Z
:language: en
:technical-committee: CALSPAM
:draft:
:toc:
:local-cache-only:
:data-uri-image:
````

generated the following HTML, with no EDITORS/AUTHORS:

````
<div class="wrapper-top">

  <div class="coverpage-doc-identity">
    <div class="doc-number">
      <span class="docnumber">CC/R/DS 18003:2018</span>
      <!-- <span class="docnumber-separator">:</span>
      <span class="docyear">2018</span> -->
    </div>

    <div class="coverpage-title">
      <span class="title-first">Calendar operator practices — Guidelines to protect against calendar abuse</span>
      <!--<span class="title-second"></span>-->
    </div>
  </div>

  <div class="coverpage-logo">
    <span>The Calendaring and Scheduling Consortium</span>
  </div>

  <div class="coverpage-tc-name">
    <span>TC CALSPAM</span>
  </div>

  
  
  </div>
````

So I need the file from you that is generating this.

**Comment by wsdwl** [comment]
The screenshot has been made from here: https://www.calendarstandards.org/standards/csd-calspam-bcp.html
This is created using the xml export from here https://github.com/CalConnect/published-csd/blob/65c64c260f9bedbb7bf7940e7915cfa7db11e5ee/csd-calspam-bcp.xml which includes 
```<contributor>
    <role type="author"/>
    <organization>
      <name>CalConnect</name>
    </organization>
  </contributor>```

So I tried to add ```:author: Thomas Schäfer,1&1 Mail&Media Development and Technology GmbH```in the adoc and generated the xml again, but that did not change the shown part above.

**Comment by opoudjis** [comment]
Indeed the author tag has not been implemented to break out the affiliation, and affiliation should be broken out. I was not informed this was a requirement, and that is going to be a separate ticket. @ronaldtse, please create and prioritise that ticket, along with how you expect affiliations to be rendered: just name, affiliation, or something else? 

I have *not* been able to replicate this issue with the header provided: the author does not appear, but neither does an empty EDITOR/AUTHOR header. I am not involved in the CSD workflow, so I do not know where the documents are; for future bug reports, please include a link to the Asciidoctor source. I have worked out here that it is github.com/CalConnect/csd-calspam-bcp

**Comment by opoudjis** [comment]
Second thing to be aware of when using the Metanorma gems: 

*They are updated all the time*.

The bug reported here is https://github.com/riboseinc/metanorma-csd/issues/69, and the metanorma-csd gem was republished with the fix three days ago. That's why I can't reproduce it now. @wsdwl, you will need to update your metanorma stack with "bundle update". 

The other issue is that @wsdwl, you were using `:author:`. The attribute implemented is `:fullname:`, not `:author:`, which is not defined in CSD Metanorma (hence why it was ignored). Please consult https://github.com/riboseinc/metanorma-csd#attributs-specific-to-csd 

This is how the Asciidoc currently renders with the updated gem, and with `fullname` used in the document instead of `author`:

[a.html 2.zip](https://github.com/riboseinc/metanorma-csd/files/2679541/a.html.2.zip)

The `&1 &` included in the document attribute is rendering as-is, and indeed if those are meant to be formatting, formatting is currently ignored in Asciidoctor document attributes.

The request to include affiliation information is separate, and I will need more information from @ronaldtse. https://github.com/riboseinc/metanorma-csd/issues/71

**Comment by ronaldtse** [comment]
I can confirm that this is fixed, with the latest metanorma-csd 1.0.12 (that's only 3 days old!).

I've pushed the relevant fixes to https://github.com/CalConnect/csd-calspam-bcp/commit/81114e8f32c265af270a65f885af0a5eac2297f4 and https://github.com/CalConnect/csd-calspam-bcp/commit/d4c516eec0f4234ea44ec2cfc2f6c56239430d7a .

**Comment by ronaldtse** [comment]
P.S. @opoudjis it is indeed 1&1 ("1und1")
